blueos_repository/consolidate.py

```diff
@@ -36,14 +44,23 @@ def from_json(json_dict: Dict[str, str]) -> "Author":
 
 
 @dataclasses.dataclass
-class Company:
+class Maintainer:
```

Comment:
This should be in a separated commit since is not connected with the label feature

Reply:
Yes it is - the old `company` label is changing to `maintainer`, which is why this code is being changed.

Reply:
Hmmm, because it’s defined at the GitHub account (user/org) level I don’t think that’s as important to change, but we could change it to `maintainer_logo` for consistency and to make it a bit more intuitive that it’s not just for companies.

Reply:
I made the `maintainer_logo` change.